resources/language/resource.language.fr_fr/strings.po

```diff
@@ -150,7 +150,7 @@ msgstr "Traceur"
 
 msgctxt "#30033"
 msgid "Use Dolby Sound"
-msgstr "Activer le son Dolby"
+msgstr "Activer le son Dolby (> 2.1)"
```

Comment:
do not enter number of channels, even 2.0 could be dolby, then in the future will be integrated in the settings a dedicated description such as kodi settings